File: TaskManagementSystem.API/TaskManagementSystem.DTO/Task/TaskRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TaskManagementSystem.Data.Enums;

namespace TaskManagementSystem.Dto.Task
{
    public class TaskRequestDto
    {
        public string Subject { get; set; }
        public int ProjectId { get; set; }
        public int? AssignTo { get; set; }
        public Priority Priority { get; set; }
        public TaskType Type { get; set; }
        public DateTime? AssignDate { get; set; }
        public DateTime? DueDate { get; set; }
        public string? Description { get; set; }
        public string? Attachment { get; set; }
        public Status? Status { get; set; }
        public bool IsActive { get; set; }
    }
}

File: TaskManagementSystem.Angular/TaskManagementSystem/src/app/shared/models/task/TaskRequestDto.ts

export interface TaskRequestDto {
    id: number | null;
    subject: string;
    projectId: number;
    assignTo: number | null;
    priority: number;
    type: number;
    assignDate: Date | null;
    dueDate: Date | null;
    description: string | null;
    attachment: string;
    status: number | null;
    isActive: boolean;
}